src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -13,6 +13,8 @@
 public class PersonCard extends UiPart<Region> {
 
     private static final String FXML = "PersonListCard.fxml";
+    private static final String[] TAG_COLOR_STYLES =
+            { "red", "blue", "orange", "black" };
```

Comment:
instead of keeping an array of strings, maybe you can create a new enum class with RED, BLUE, ORANGE and BLACK